android/java/org/chromium/chrome/browser/crypto_wallet/permission/BraveEthereumPermissionPromptDialog.java

```diff
@@ -96,6 +96,9 @@ public BraveEthereumPermissionPromptDialog(long nativeDialogController,
         }
     }
 
+    // TODO: It is much more efficient to retrieve resources by identifier (e.g. R.foo.bar) than by
```

Comment:
is there a ticket for this?

Reply:
Created one https://github.com/brave/brave-browser/issues/25388